astropy/stats/funcs.py

```diff
@@ -716,7 +720,7 @@ def poisson_conf_interval(n, interval='root-n', sigma=1, background=0,
     return conf_interval
 
 
-def median_absolute_deviation(a, axis=None):
+def median_absolute_deviation(data, axis=None, ignore_nan=False):
```

Comment:
If you're going to rename the parameter #5214 would be ideal here :smile: 

(sorry for double posting this)